src/PKSim.Core/Model/PopulationSimulation.cs

```diff
@@ -270,37 +271,41 @@ public override void UpdateFromOriginalSimulation(Simulation originalSimulation)
          sourcePopSimulation?.AdvancedParameters.Each(x => AddAdvancedParameter(x, generateRandomValues: true));
       }
 
-      private DataColumn aggregateDataColumns(IReadOnlyList<DataColumn> columns)
+      private DataColumn medianAggregateDataColumns(IReadOnlyList<DataColumn> columns)
       {
          var column = columns?.FirstOrDefault();
          if (column == null)
             return null;
 
-         return new DataColumn(column.Id, column.Dimension, column.BaseGrid)
+         return  new DataColumn(column.Id, column.Dimension, column.BaseGrid)
          {
-            Values = Enumerable.Range(0, column.Values.Count)
-               .Select(i => 
-                  columns.Select(x => x.Values[i]).ToList().ArithmeticMean()
+            Values = Enumerable.Range(0, column.Values.Count).Select(i => 
```

Comment:
going through the data columns and taking the nth value in each case and then aggregating the median. There's no shortcut obvious to me. But the inside linq stuff can be hidden a bit by a method